### PR TITLE
PIC-4153 upgrade s3

### PR DESCRIPTION
**Pull request by markreesmoj**

### CODE REVIEW

**Comment by markreesmoj** [comment]
To get rid of this warning

024-09-20 13:25:34.658  WARN 1 --- [           main] com.amazonaws.util.VersionInfoUtils      : The AWS SDK for Java 1.x entered maintenance mode starting July 31, 2024 and will reach end of support on December 31, 2025. For more information, see https://aws.amazon.com/blogs/developer/the-aws-sdk-for-java-1-x-is-in-maintenance-mode-effective-july-31-2024/
You can print where on the file system the AWS SDK for Java 1.x core runtime is located by setting the AWS_JAVA_V1_PRINT_LOCATION environment variable or aws.java.v1.printLocation system property to 'true'.
This message can be disabled by setting the AWS_JAVA_V1_DISABLE_DEPRECATION_ANNOUNCEMENT environment variable or aws.java.v1.disableDeprecationAnnouncement system property to 'true'.
The AWS SDK for Java 1.x is being used here:

**Review by johnyfarrar — APPROVED** [review]
LGTM

File: src/test/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/integration/endpoint/ExternalDocRequestEndpointIntTest.kt [hunk 1]
```diff
@@ -56,27 +60,17 @@ class ExternalDocRequestEndpointIntTest : IntegrationTestBase() {
     @BeforeEach
     fun beforeEach() {
         courtCaseEventsQueue?.sqsClient?.purgeQueue(PurgeQueueRequest.builder().queueUrl(courtCaseEventsQueue!!.queueUrl).build())
-        amazonS3.createBucket(bucketName, "eu-west-2")
+        amazonS3.createBucket(CreateBucketRequest.builder().bucket(bucketName).build())
```
Comment (Obsiye) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/integration/endpoint/ExternalDocRequestEndpointIntTest.kt:63]: Does it matter that the region "eu-west-2" is not set  as it looks like it is gone?

**Review by Obsiye — APPROVED** [review]

File: src/test/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/integration/endpoint/ExternalDocRequestEndpointIntTest.kt (hunk 1 above)
Comment (markreesmoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/integration/endpoint/ExternalDocRequestEndpointIntTest.kt:63]: It is set by default: 
https://github.com/ministryofjustice/crime-portal-gateway/blob/PIC-4153-upgrade-s3/src/test/resources/application-test.yml#L28

Comment (markreesmoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/integration/endpoint/ExternalDocRequestEndpointIntTest.kt:63]: image

Comment (markreesmoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/integration/endpoint/ExternalDocRequestEndpointIntTest.kt:63]: See this debug breakpoint showing the region set on the ObjectListing

**Pull request merged**